### DISCUSSION TITLE
tech-company-interview/

### DISCUSSION THREAD

**Discussion by giscus** [TennyZhuang/blog#27] [Announcements]
# tech-company-interview/

从大学找实习开始，断断续续面试、被面试了许多场，想来应该有三位数了，最近又作为 Senior/Leader 岗位面了一些公司，算是补全了一块人生体验，写篇文章谈谈 Tech 公司面试的感想。 算法谈到算法面试，首先想到的是 LeetCode，这不是我想的，是 Copilot 想的。  算法是讨论到面试绕不开的话题，我懒得考古，但这股“歪风”应该是从硅谷几家 Big-name Tech 带

https://blog.zhuangty.com/tech-company-interview/

**Comment by daya0576** [comment]
写的太好了，我有两个小困惑：
1. 当前的面试机制，无论大公司还是初创公司，都存在一定诟病。但为什么大多数还是以这种模式运行？并且似乎招到的候选人偏差不大。
2. 人不同于机器，软性的方面，例如性格等因素也占很重要的因素，有比较成熟的考察手段吗？

**Reply by RegulusGuo** [reply]
1 我不了解, 回答一下2
国内一些大厂 (如菊厂) 面试有性格测试环节, 问题有点像 MBTI 测试的职场情景版, 但可想而知现在求职者都会专门选那些出题人喜欢的选项了, 真实性格测不出来的.

**Reply by GrayXu** [reply]
主要还是面试成本问题吧

**Reply by TennyZhuang** [reply]
谈谈个人理解

> 当前的面试机制，无论大公司还是初创公司，都存在一定诟病。但为什么大多数还是以这种模式运行？并且似乎招到的候选人偏差不大。

首先这种模式对大公司有好处的，大公司愿意用，而中小公司更多是跟风，其实这种面试方法不适合他们。他们并没有充足的人才库，也不是招到够用的螺丝钉就行，其实他们更应该用更多元化的面试方式来挑出被大厂面试流程误杀的人才。

> 人不同于机器，软性的方面，例如性格等因素也占很重要的因素，有比较成熟的考察手段吗？

这值得另一篇文章了，感觉我的理解也还不够到位，等我想明白了再来写。

**Comment by daya0576** [comment]
还是如同你说的，大部分场景下，公司招的是螺丝钉而不是人生伴侣，所以当前主流的招聘模式是行得通的

**Reply by andyhe007** [reply]
大佬好，我是量化猎头andy，专注量化行业两年帮助20多个人才入职新公司，看您在代码方面比较厉害想给你分享一些量化行业私募机会，希望可以交个朋友。微信：quant_andyhe1
